src/alpine/gp/primitives.py

```diff
@@ -147,3 +147,53 @@ def add_primitives_to_pset(
                     in_types = primitives_collection[typed_primitive].in_types
                     out_type = primitives_collection[typed_primitive].out_type
                     pset.addPrimitive(op, in_types, out_type, name=typed_primitive)
+
+
+def deap_primitive_to_sympy_expr(prim: Primitive, conversion_rules: Dict, args: Tuple):
+    """Convert a deap scalar primitive into a sympy primitive.
```

Comment:
Convert a DEAP primitive and its arguments into the corresponding sympy expression.